chess/svg.py

```diff
@@ -58,6 +59,13 @@
 }}
 """.format(**DEFAULT_COLORS)
 
+class Arrow():
```

Comment:
How about making this a `namedtuple` like so: https://github.com/niklasf/python-chess/blob/master/chess/uci.py#L69? And then allowing the more succint `board(..., arrows=[(E2, E4)])` would be nice.